collector/generatorreceiver/internal/topology/service_route.go

```diff
@@ -33,27 +33,45 @@ func (r *ServiceRoute) validate(t Topology) error {
 		}
 	}
 
-	if r.LatencyPercentiles == nil && r.MaxLatencyMillis <= 0 {
+	if r.LatencyConfigs == nil && r.MaxLatencyMillis <= 0 {
 		return fmt.Errorf("must have either latencyPercentiles or positive, non-zero maxLatencyMillis defined")
 	}
 	return nil
 }
 
 func (r *ServiceRoute) load(route string) error {
 	r.Route = route
-	if r.LatencyPercentiles != nil {
-		err := r.LatencyPercentiles.loadDurations()
+	if r.LatencyConfigs == nil {
+		if r.MaxLatencyMillis == 0 {
+			return fmt.Errorf("route must include maxLatencyMillis or latencyConfigs")
+		} else {
+			return nil
+		}
+	}
+	hasDefault := false
+	for _, cfg := range r.LatencyConfigs {
+		err := cfg.loadDurations()
 		if err != nil {
 			return fmt.Errorf("error parsing latencyPercentiles: %v", err)
 		}
+		err = cfg.ValidateFlags()
+		if err != nil {
+			return err
+		}
+		if cfg.IsDefault() {
+			hasDefault = true
```

Comment:
Should we also return an error if it contains multiple default configs? (since in `LatencyConfigs.Sample()` they'd end up getting overwritten by whichever default config appears last)

Reply:
Good point! That could definitely be confusing.